pkg/api.go

```diff
@@ -92,7 +92,10 @@ type Check struct {
 	Severity     string            `json:"severity"`
 	Icon         string            `json:"icon"`
 	DisplayType  string            `json:"displayType"`
-	CheckCanary  *v1.Canary        `json:"-"`
+	RunnerName   string            `json:"runnerName"`
+	// Specify the canary location, <runner>/<namespace>/<name>
+	Location string     `json:"location"`
```

Comment:
What is the different between runner name and location ?

Reply:
So the runner name is just the name of the runner. The Location would be <runner-name>/<namespace>/<name>. I added this so we don't end up with nested loops to fetch the canary object, with these parameters